charybdis/disasm.py

```diff
@@ -259,5 +271,31 @@ def decode_insn(rom: bytes, index: int) -> Optional[DecodedInsn]:
         # LD A, A
         case 0x7f:
             decoded = insn.Insn(name=insn.InsnName.LD, operands=[insn.R8.A, insn.R8.A])
+        # LD (n), A
+        case 0xe0:
+            size = 2
+            offset = rom[index + 1]
+            decoded = insn.Insn(name=insn.InsnName.LDH, operands=[insn.DirectU8(offset), insn.R8.A])
+        # LD (C), A
+        case 0xe2:
+            decoded = insn.Insn(name=insn.InsnName.LDH, operands=[insn.IndirectR8(insn.R8.C), insn.R8.A])
+        # LD (nn), A
+        case 0xea:
+            size = 3
+            addr = rom[index + 1] + (rom[index + 2] << 8)
+            decoded = insn.Insn(name=insn.InsnName.LD, operands=[insn.DirectU16(insn.U16(addr)), insn.R8.A])
+        # LD A, (n)
+        case 0xf0:
+            size = 2
+            offset = rom[index + 1]
+            decoded = insn.Insn(name=insn.InsnName.LDH, operands=[insn.R8.A, insn.DirectU8(offset)])
+        # LD A, (C)
+        case 0xf2:
+            decoded = insn.Insn(name=insn.InsnName.LDH, operands=[insn.R8.A, insn.IndirectR8(insn.R8.C)])
+        # LD A, (nn)
```

Comment:
I think we have been a bit inconsistent when referring to indirection, sometimes writing `(...)`, other times `[...]`. Let's standardize on using square brackets.